ec/jacobian_t.hpp

```diff
@@ -538,5 +526,22 @@ template<class field_t> class jacobian_t {
         }
         *this = p3;
     }
+
+#ifndef NDEBUG
+    friend inline bool operator==(const jacobian_t& a, const jacobian_t& b)
+    {   return (a.X == b.X) && (a.Y == b.Y) && (a.Z == b.Z);   }
```

Comment:
This is not how you compare points in Jacobian coordinates for equality. See [POINT_IS_EQUAL_IMPL](https://github.com/supranational/blst/blob/master/src/ec_ops.h#L467-L488) for a template. Same naturally applies to the inequality test, which should read just `!(a == b)`. Or maybe compiler does it for you without having to define `operator!=`?

Reply:
> This is not how you compare points in Jacobian coordinates for equality.

Right, same thing with xyzz_t. I'll fix both.